Update releasenotes.md (#888)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/releasenotes.md

[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

   > [!IMPORTANT]
    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
#### General
* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.

#### .NET  

*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
*   Updated NuGet Target to include WebViewLoader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
*   Updated WebResourceRequested to expose HttpRequestMessage and HttpResponseMessage APIs in .NET.  \([\#131][GithubMicrosoftedgeWebviewfeedbackIssue131]\).

## 0.9.622  
Release Date: Sept 09, 2020

[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

#### General  

*   > [!IMPORTANT]

>     > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 

Reviewer: I would say "SDK 0.9.622 is the release candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality."

*   > [!IMPORTANT]

>     > **Announcement**:  Visual Hosting is now in preview.

Reviewer: Visual hosting in preview in Pre-release. You probably want to also briefly explain what visual hosting is and give some pointers. You can work with either Brad or me on that.

# Release notes for WebView2 SDK  

The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a six week cadence.  Review the following content for up-to-date information about product announcements, additions and modifications to the API surface, and breaking changes.  

> [!NOTE]
> Re-compile your app after updating the NuGet package.  

> [!IMPORTANT]
> While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  

## 0.9.628-prerelease  
Release Date: Sept 09, 2020

> [NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

Reviewer 2: Please update release date as it is postponed until today, add two nuget package entries, and confirm the minimum supported version.

   > [!IMPORTANT]
    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.

#### General
*   > [!IMPORTANT]
    > **Announcement**:  Visual Hosting is now in preview.

> * Added the CompositionController as an experimental interface. This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.

Reviewer 2: ```suggestion
* Added the CompositionController as an experimental interface. This lets your app connect WebView2 to DComp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.
```

#### .NET  

> *   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).

Reviewer 2: ```suggestion
*   .NET Binaries are now [strongly-named](https://docs.microsoft.com/dotnet/standard/assembly/strong-named).  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
```

> *   Updated NuGet Target to include WebViewLoader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).

Reviewer 2: ```suggestion
*   Updated NuGet Target to include WebView2Loader.dll.  \([\#228][GithubMicrosoftedgeWebviewfeedbackIssue228]\) & \([\#183][GithubMicrosoftedgeWebviewfeedbackIssue183]\).
```

*   Updated WebResourceRequested to expose HttpRequestMessage and HttpResponseMessage APIs in .NET.  \([\#131][GithubMicrosoftedgeWebviewfeedbackIssue131]\).

## 0.9.622  
Release Date: Sept 09, 2020

> [NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

Reviewer 2: Same as above.

#### General  

*   > [!IMPORTANT]
    > **Announcement**: This SDK is the Release Candidate for WebView2 Win32 C/C++ GA. The GA version will likely have the same API interface and functionality.

> *   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 

Reviewer 2: Avoid hard-coded locale code in the links. The warnings can be found in the Docs Build Status.
```suggestion
*   Disconnected [browser policies](https://docs.microsoft.com/deployedge/microsoft-edge-policies). 
```

> *   Added [AllowSingleSignOnUsingOSPrimaryAccount][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.

Reviewer 2: The path for AllowSingleSignOnUsingOSPrimaryAccount would be `./reference/win32/0-9-622/icorewebview2environmentoptions.md#get_allowsinglesignonusingosprimaryaccount`

> *   Updated ICoreWebView2NewWindowRequestedEventArgs to include WindowFeatures property, and its associated ICoreWebView2WindowFeatures.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).

Reviewer 2: Link ICoreWebView2NewWindowRequestedEventArgs  to `./reference/win32/0-9-622/icorewebview2newwindowrequestedeventargs.md`.

*   Updated System.Windows.Rect. to use System.Drawing.Rectangle instead of System.Windows.Rect \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).

> *   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).

Reviewer 2: "Updated NewWindowRequested event to handle window.open() call without parameters"?

> *   [AdditionalBrowserArguments](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/icorewebview2environmentoptions#put_additionalbrowserarguments) set via ICoreWebView2EnvironmentOptions will not be overridden with environment variable nor registry value. See [CreateCoreWebView2EnvironmentWithOptions](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/webview2-idl#createcorewebview2environmentwithoptions) for more details.

Reviewer 2: ```suggestion
*   [AdditionalBrowserArguments](./reference/win32/0-9-622/icorewebview2environmentoptions#put_additionalbrowserarguments) set via ICoreWebView2EnvironmentOptions will not be overridden with environment variable nor registry value. See [CreateCoreWebView2EnvironmentWithOptions](./reference/win32/0-9-622/webview2-idl#createcorewebview2environmentwithoptions) for more details.
```

## 0.9.579  

> Release Date: July 20, 2020

Reviewer 2: Shall we change all date formats?

# Release notes for WebView2 SDK  

The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a six week cadence.  Review the following content for up-to-date information about product announcements, additions and modifications to the API surface, and breaking changes.  

> [!NOTE]
> Re-compile your app after updating the NuGet package.  

> [!IMPORTANT]
> While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  

## 0.9.628-prerelease  

Release Date: Sept 09, 2020  

[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

> [!IMPORTANT]
> **Announcement**:  This prerelease continues to release out of Microsoft Edge 87 Branch.  In the future, pre-release SDK releases match the Edge Canary builds, whereas the normal releases are synchronized with Edge Stable.  

#### General  

*   > [!IMPORTANT]
    > **Announcement**:  Visual Hosting is now in preview.  
*   Added the CompositionController as an experimental interface.  This lets your app connect WebView2 to Dcomp visual or WinComp ContainerVisual and provides API that allows the app to forward mouse and touch inputs to the WebView.  

#### .NET  

> *   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).  

Reviewer 3: Please provide a link to the `Strongly Named` article.

> [!IMPORTANT]
> While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  

## 0.9.627-prerelease  
Release Date: 09/08/2020

[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

   > [!IMPORTANT]
    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.

#### General  
* 
#### .NET  

*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).

## 0.9.622  

> Release Date: 09/08/2020

Reviewer 2: Release date would most likely be next Wednesday. Adding a comment here to ensure we don't forget to update it :)

> [NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  

Reviewer 2: Check with @pagoe-msft to see whether we should use WebView2 Runtime version for release packages and Canary version for prerelease packages.

Author: @pagoe-msft Do you have insight into Jessica's comment above

#### General  

*   > [!IMPORTANT]
    > **Announcement**: This SDK release is a candidate for WebView2 Win32 C++ GA. 

> * > [!IMPORTANT]

Reviewer 2: ```suggestion
*   > [!IMPORTANT]
```

    > **Announcement**:  Visual Hosting is now in preview.

*   Disconnected [browser policies](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-policies). 

> *   Added [IsSingleSignOnUsingOSPrimaryAccountEnabled][ReferenceWin3209538Icorewebview2experimentaloptionsGetIssinglesignonusingosprimaryaccountenabled] property on WebView2 environment options to enable conditional access for WebView.

Reviewer 2: The API is renamed to `AllowSingleSignOnUsingOSPrimaryAccount` and link to the doc is TBD.

*   Added 
*   Fixed System.Windows.Rect.  \([\#235][GithubMicrosoftedgeWebviewfeedbackIssue235]\).
*   Updated the Windows.open() function to handle "no parameter" case.  \([\#293][GithubMicrosoftedgeWebviewfeedbackIssue293]\).

> #### .NET  

Reviewer 2: These .NET items should be under 0.9.627-prerelease. There will be no .NET bits in 0.9.622.
